app/templates/components/upf-table/header-cell.hbs

```diff
@@ -5,7 +5,7 @@
 {{/if}}
 
 {{#if column.tooltip}}
-  <fa class="fa fa-info-circle" data-toggle="tooltip" title={{column.tooltip}}></fa>
+  <OSS::Icon @icon="fa-info-circle" @style="solid" {{enable-tooltip title=column.tooltip placement="top"}} />
 {{/if}}
```

Comment:
given the code, this is totally dead. but let's keep it for safety for now

Reply:
It is not used in the header cell table in acquisition? Look up my screenshot